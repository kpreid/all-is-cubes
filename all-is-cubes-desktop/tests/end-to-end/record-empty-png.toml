# It is not an error if there is nothing to record. (But perhaps it should be?)
bin.name = "all-is-cubes"
args = "--no-config-files --template=blank --graphics=record --output=test-record-empty.png"
status.code = 0
