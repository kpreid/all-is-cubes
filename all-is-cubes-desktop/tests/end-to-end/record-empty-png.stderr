...

Wrote test-record-empty.png
Flaws in recording: NO_BLOOM
